src/main/java/de/unistuttgart/iste/sqa/pse/sheet11/presence/calculator/BuggyCalculator.java

```diff
@@ -32,15 +32,12 @@ public int divide(final int dividend, final int divisor) {
 		return posQuotient * getResultSign(dividend, divisor);
 	}
 	
-	/*
-	 * @ensures dividend < 0 & divisor < 0 ==> \result = 1
-	 * 
-	 * @ensures dividend > 0 & divisor > 0 ==> \result = 1
-	 * 
-	 * @ensures dividend < 0 & divisor > 0 ==> \result = -1
-	 * 
-	 * @ensures dividend > 0 & divisor < 0 ==> \result = -1
-	 */
+	/*@
+	 @ ensures dividend < 0 & divisor < 0 ==> \result == 1
```

Comment:
- [ ] Logical "and" should be a double `&&` 
- [ ] I would recommend joining the single postconditions with `&&` to have ONE "ensures" statement
- [ ] JML lines require a semicolon at the end

Reply:
There are still & in this JML formula